pkg/kubelet/kubelet.go

```diff
@@ -1875,14 +1872,9 @@ func (kl *Kubelet) syncLoop(updates <-chan PodUpdate, handler SyncHandler) {
 	glog.Info("Starting kubelet main sync loop.")
 	var housekeepingTimestamp time.Time
 	for {
-		if !kl.containerRuntimeUp() {
-			time.Sleep(5 * time.Second)
-			glog.Infof("Skipping pod synchronization, container runtime is not up.")
-			continue
-		}
-		if !kl.doneNetworkConfigure() {
+		if rs := kl.runtimeState.errors(); len(rs) != 0 {
```

Comment:
This seems semantically different from previous code because runtimeState would start with no errors. If syncNetworkStatus hasn't run even once, we'd assume the network is configured?